src/types/siteLaunch.ts

```diff
@@ -0,0 +1,46 @@
+export interface DNSRecord {
+  source: string
+  type: string
+  target: string
+}
+
+export type SiteLaunchFrontEndStatus =
+  | "LAUNCHED"
```

Comment:
nit: Can expose as object first?

```
const SiteLaunchStatus = {
  Launched: "LAUNCHED",
  NotLaunched: "NOT_LAUNCHED",
  Launching: "LAUNCHING"
ChecklistTasksPending: "CHECKLIST_TASKS_PENDING
} as const;

type SiteLaunchFrontEndStatus = typeof SiteLaunchStatus[keyof typeof SiteLaunchStatus]
```

Reply:
Hey is there a reason why you prefer this? A bit inclined not to do this as there is a semantic difference between the front end site launch status vs the backend ones since some of the state is solely held in the front end. 

am ok to change it to object but do you have suggestions for some-var-name where it is clear it only refers to the states in front end? 
```
const <some-var-name> = {
  Launched: "LAUNCHED",
  NotLaunched: "NOT_LAUNCHED",
  Launching: "LAUNCHING"
ChecklistTasksPending: "CHECKLIST_TASKS_PENDING
} as const;

type SiteLaunchFrontEndStatus = typeof SiteLaunchStatus[keyof typeof some-var-name]
```

Reply:
Sure we can use something like `SiteLaunchFrontEndStatusOptions` and `type SiteLaunchFrontEndStatus` which is one of those options?

Reply:
sure, 38945a2083e6b0c7d7043ce3711449d837da7889